main/build.gradle.kts

```diff
@@ -6,11 +6,8 @@
 plugins {
     id("com.android.application")
     id("checkstyle")
-}
-
-apply {
-    plugin("kotlin-android")
-    plugin("kotlin-android-extensions")
+    kotlin("android")
+    kotlin("android.extensions")
 }
```

Comment:
Could you elaborate a bit more why this change should be done? Is it style, does it fix something, does it improve something?

Reply:
To make it convenient syntax. :)

Reply:
To make the command line easier and shorter and add the CheckstyleExtension again so that I know what note to do better

Reply:
The commit messages themselves are just "minor change in build.gradle.kts"